docs/analytics/what-is-amplitude.md

```diff
@@ -2,145 +2,54 @@
 title: What is Amplitude?
 description: Amplitude Analytics is the leading product analytics tool. It helps you gather and democratize data about how users engage with your apps.
 ---
+<script src="https://fast.wistia.com/embed/medias/g704qzqk6e.jsonp" async></script><script src="https://fast.wistia.com/assets/external/E-v1.js" async></script><div class="wistia_responsive_padding" style="padding:56.25% 0 0 0;position:relative;"><div class="wistia_responsive_wrapper" style="height:100%;left:0;position:absolute;top:0;width:100%;"><div class="wistia_embed wistia_async_g704qzqk6e videoFoam=true" style="height:100%;position:relative;width:100%"><div class="wistia_swatch" style="height:100%;left:0;opacity:0;overflow:hidden;position:absolute;top:0;transition:opacity 200ms;width:100%;"><img src="https://fast.wistia.com/embed/medias/g704qzqk6e/swatch" style="filter:blur(5px);height:100%;object-fit:contain;width:100%;" alt="" aria-hidden="true" onload="this.parentNode.style.opacity=1;" /></div></div></div></div>
 
-To understand how Amplitude works, let’s walk through a hypothetical example.
+Amplitude is a powerful product analytics platform that enables you to build better products by tracking and understanding user behavior. 
 
-Tunes is a standard music player for mobile devices that has common actions like playing a song, skipping a song, shuffling play, and sharing a song.
+With Amplitude, you can track user data and gain insights into user engagement, retention, and revenue. Amplitude keeps your data trustworthy and secure, ensuring that you have access to accurate and reliable information. Amplitude offers powerful analytics tools that help answer questions about what happened, why it happened, and which actions to take next. With Amplitude, you can seamlessly share your work across teams, facilitating collaboration and driving growth.
 
-Using Amplitude, you can track all the actions your users make in detail and better understand what’s working and what’s not.
+## Key concepts
 
-### What to track
+Amplitude is an event-based analytics tool that tracks the behaviors of users based on in-product interactions and analyzes user behavior in real-time. Event-based analytics is the method of tracking and analyzing interactions between users and your product, also known as events. Before you get started with Amplitude, it's important to know the key concepts used in Amplitude. 
 
-Amplitude gives you the power to decide what actions you want to track. You can choose to track anything and everything.
+|Name|Description|A music player app example|
+|---|-----------|-----------|
+|Users|A user is the specific individual that interacts with your product|A user uses the app to play music|
```

Comment:
I think the order should be 
Events, Users, Event Props, User Props
OR
Events, Event Props, Users, User Props

Reply:
It makes sense to put events at the first place as it is the key of event-base analysis and also put event properties close to events. 

Changed it to Events, Event Props, Users, User Props